src/docker-osx-dev

```diff
@@ -375,17 +381,79 @@ function init_boot2docker {
     log_info "Starting Boot2Docker VM"
     boot2docker start --vbox-share=disable
   fi
+
+  configure_boot2docker
+}
+
+################################################################################
+# docker-machine manipulation
+################################################################################
+
+#
+# Read a json string from stdin and return the value of the key provided as first argument
+# Uses head -n 1 to avoid duplicates
```

Comment:
This is an implementation detail. Instead, you should probably say something along the lines of "Does a simple string search and returns the first string value that matches the given key".